roles/kubernetes/control-plane/tasks/psp-install.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Check AppArmor status
+  command: which apparmor_parser
+  register: apparmor_status
+  when:
+    - podsecuritypolicy_enabled
+    - inventory_hostname == groups['kube_control_plane'][0]
```

Comment:
This new psp-install.yml is used when
```
    - podsecuritypolicy_enabled
    - inventory_hostname == groups['kube_control_plane'][0]
```
from kubeadm-setup.yml.
The above conditions seem to be removed for cleanup.
The same comment can be applied to another lines.

Reply:
Sure i can remove them :+1: